drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -451,26 +563,40 @@ private String getFileDownloadURL(final Supplier<Path> pathSupplier) {
         return this::getModelSuccessCallbackMethod;
     }
 
-    private Optional<RightPanelView> getRightPanelView() {
-        final DefaultPlaceRequest placeRequest = new DefaultPlaceRequest(RightPanelPresenter.IDENTIFIER);
-        if (PlaceStatus.OPEN.equals(placeManager.getStatus(placeRequest))) {
-            final AbstractWorkbenchActivity rightPanelActivity = (AbstractWorkbenchActivity) placeManager.getActivity(placeRequest);
-            return Optional.of((RightPanelView) rightPanelActivity.getWidget());
+    private Optional<TestToolsView> getTestToolsView(PlaceRequest placeRequest) {
+        final Activity activity = placeManager.getActivity(placeRequest);
+        if (activity != null) {
+//        if (PlaceStatus.OPEN.equals(placeManager.getStatus(placeRequest))) {
```

Comment:
Is `placeManager.getActivity(placeRequest)` equivalent to `PlaceStatus.OPEN.equals(placeManager.getStatus(placeRequest))`?
(btw commented code to remove)